doc/en/components/grids/_shared/export-excel.md

```diff
@@ -136,26 +136,48 @@ public webGridExportEventFreezeHeaders(args: any): void {
 <!-- end: WebComponents -->
 
 ```razor
- <IgbGrid>
+ <{ComponentSelector}>
     <IgbGridToolbar>
       <IgbGridToolbarActions>
         <IgbGridToolbarExporter
-          ExportExcel="true" ExportStarted="WebGridExportEventMultiColumnHeaders">
+          ExportExcel="true" ExportStartedScript="WebGridExportEventMultiColumnHeaders">
         </IgbGridToolbarExporter>
       </IgbGridToolbarActions>
     </IgbGridToolbar>
- </IgbGrid>
+ </{ComponentSelector}>
 
 @code {
 
   public void WebGridExportEventMultiColumnHeaders(IgbExporterEventEventArgs args)
   {
-      bool exportMultiHeaders = (bool)exportHeaders.GetCurrentPrimitiveValue();
       args.Detail.Options.FreezeHeaders = true;
   }
 }
 ```
 
+<!-- ComponentStart: TreeGrid -->
+```razor
+ <{ComponentSelector}>
+    <IgbGridToolbar>
+      <IgbGridToolbarActions>
+        <IgbGridToolbarExporter
+          ExportExcel="true" ExportStartedScript="WebGridExportEventMultiColumnHeaders">
```

Comment:
Should probably be `ExportStarted` since the snippet below is the server event.

Reply:
I changed this because the server side event does not fire. https://infragistics.visualstudio.com/NetAdvantage/_git/code-generation-library/pullrequest/9569?_a=files
Here is the PR. Will update the snippet